tests/test_convert_lit_checkpoint.py

```diff
@@ -0,0 +1,78 @@
+from pathlib import Path
+from urllib.request import urlretrieve
+
+import pytest
+import torch
+
+wd = Path(__file__).parent.parent.absolute()
+
+
+@torch.inference_mode()
+def test_against_original_falcon_40b():
+    file_path = wd / "tests" / "original_falcon_40b.py"
+    url = "https://gist.githubusercontent.com/carmocca/feed39b1bc65a29f73c1cecc58a01167/raw/a9a65f2b93716b3c09ec9f354d535ae5953de08f/original_falcon_40b.py"
+    if not file_path.is_file():
+        urlretrieve(url=url, filename=file_path)
+
+    from tests.original_falcon_40b import RWConfig, RWForCausalLM
+    from lit_gpt import Config, GPT
+    from scripts.convert_lit_checkpoint import copy_weights_falcon
+    from scripts.convert_hf_checkpoint import copy_weights_falcon as copy_weights_falcon_hf
```

Comment:
This would make the code easier to read IMO

```suggestion
    from scripts.convert_lit_checkpoint import copy_weights_falcon as copy_to_theirs
    from scripts.convert_hf_checkpoint import copy_weights_falcon as copy_to_ours
```